env-sample

```diff
@@ -0,0 +1,4 @@
+AWS_SECRET_ACCESS_KEY=
```

Comment:
Are these supposed to be filled out?

Reply:
this is the example file. The real one is in the gitignore for security